build-tools/build-indexes

```diff
@@ -1046,7 +1102,7 @@ process.stdout.write("Building `data/teambuilder-tables.js`... ");
 	// Mods
 	//
 
-	for (const mod of ['gen7letsgo', 'gen8bdsp']) {
+	for (const mod of ['gen7letsgo', 'gen8bdsp', 'alternatium', 'randomdex']) {
```

Comment:
this is more server-side, but we should probably make these mod files pmots and pmlc to make support consistent in future cases